core/core.iml

```diff
@@ -10,6 +10,20 @@
     <orderEntry type="jdk" jdkName="1.6" jdkType="JavaSDK" />
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="library" scope="TEST" name="junit:junit:4.12" level="application" />
-    <orderEntry type="library" scope="TEST" name="org.hamcrest:hamcrest-library:1.3" level="application" />
+    <orderEntry type="library" scope="TEST" name="org.hamcrest:hamcrest-all:1.3" level="application" />
+    <orderEntry type="library" name="lib" level="project" />
+    <orderEntry type="library" name="tomcat-coyote-util" level="project" />
+    <orderEntry type="library" name="start" level="project" />
+    <orderEntry type="library" name="javax.servlet.jsp-2.2.0.v201112011158" level="project" />
+    <orderEntry type="module-library" scope="TEST">
+      <library>
+        <CLASSES>
+          <root url="jar://$MODULE_DIR$/java/test/junit/net/i2p/junit-4.12.jar!/" />
```

Comment:
This line (21), and the one below it (22), are examples of where a reference to JUnit and hamcrest were left out (no .jar files or supporting libraries of any form were linked, so matchers were not working and tests wouldn't run).